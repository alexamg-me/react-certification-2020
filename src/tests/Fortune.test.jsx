import React from 'react';
import TestRenderer from 'react-test-renderer';
import Fortune from '../components/Fortune/Fortune.component';

describe(`Component: Fortune`, () => {
  test(`Fortune renders with default props`, () => {
    const component = TestRenderer.create(<Fortune />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
