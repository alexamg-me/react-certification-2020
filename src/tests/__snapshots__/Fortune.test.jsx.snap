// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Fortune Fortune renders with default props 1`] = `
<span
  className="fortune"
>
   
</span>
`;
